src/Weasel.Postgresql/Tables/NullsSortOrder.cs

```diff
@@ -0,0 +1,13 @@
+namespace Weasel.Postgresql.Tables
+{
+    /// <summary>
+    /// Specifies the null sort order
+    /// </summary>
+    public enum NullsSortOrder
+    {
```

Comment:
**SUGGESTION:** We could add `None`, or `NotDefined` option and use it in the code, then it'd be more explicit than null.

Reply:
Added `None`. Fixed and handled this suggestion.